packages/components/Input/src/InputTokens.ios.ts

```diff
@@ -0,0 +1,86 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { InputTokens } from './Input.types';
+
+export const defaultInputTokens: TokenSettings<InputTokens, Theme> = (t: Theme) =>
+  ({
+    iconSize: 24,
+    hasIcon: {
+      //spacingIconContent + iconSize
```

Comment:
Remove commented out bit?

Reply:
Have added this to explain the value (In figma it is two separate values which translate to one single here)